wombat-admin/src/components/statistics.tsx

```diff
@@ -13,7 +13,7 @@ export default function Statistics({ incomingRequests }: StatisticsProps) {
   ).length;
 
   const adminAccessRequiredRequests = incomingRequests.filter(
-    (request) => request.state === "MANUAL_OVERRIDE_REQUESTED"
+    (request) => request.state === "MANUAL_OVERRIDE_ACTIONED"
```

Comment:
Did we mean to make this change? We want only the requests where they have been actioned?